pkgs/development/python-modules/loca/default.nix

```diff
@@ -0,0 +1,25 @@
+{ lib, buildPythonPackage, pythonOlder, fetchFromSourcehut }:
+
+buildPythonPackage rec {
+  pname = "loca";
+  version = "2.0.1";
+  format = "flit";
+  disabled = pythonOlder "3.7";
+
+  src = fetchFromSourcehut {
+    owner = "~cnx";
+    repo = pname;
+    rev = version;
+    sha256 = "1l6jimw3wd81nz1jrzsfw1zzsdm0jm998xlddcqaq0h38sx69w8g";
+  };
+
+  doCheck = false; # all checks are static analyses
+  pythonImportsCheck = [ "loca" ];
+
+  meta = with lib; {
+    description = "Local locations";
+    homepage = "https://pypi.org/project/loca";
```

Comment:
They list https://sr.ht/~cnx/loca/ as the homepage on pypi, I feel like that is more correct

Reply:
I am actually the upstream maintainer, and the sr.ht project cover implementations in multiple language; the README might change to a meta repository some time in the future.  The PyPI link on the other hand will always have the relevant information, plus some metadata extracted from the wheel.

Reply:
Oh, well, nevermind then :)